tools/install-source/InstallSourcesLib/InstallSourcesLib.csproj

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Well, I added it to be able tot do a nice unit testing of the diff path manglers. There is some logic between generated code and note. I can move it to the tool and see if I can reference it in the unit tests.